src/App.tsx

```diff
@@ -1,7 +1,31 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedButton: string;
+};
+
+export class App extends React.Component {
+  state: Readonly<State> = {
+    pressedButton: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keydown', (e) => {
+      const pressedButton = e.key;
```

Comment:
```suggestion
      const newPressedButton = e.key;
```
For now your name is a bit confusing with state variable

Reply:
fixed

Reply:
this one is not fixed

Reply:
state variable was renamed as `pressedKey`, thus state variable and const in componentDidMount() have a different name